app/src/sharedTest/java/org/oppia/app/activity/InputInteractionViewTestActivityTest.kt

```diff
@@ -71,14 +74,111 @@ class InputInteractionViewTestActivityTest {
     activityScenario.onActivity { activity ->
       activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
     }
-    onView(withId(R.id.test_number_input_interaction_view)).check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
+    onView(withId(R.id.test_number_input_interaction_view)).check(matches(isDisplayed()))
+      .check(
+        matches(withText("9"))
+      )
+    activityScenario.recreate()
+    onView(withId(R.id.test_number_input_interaction_view)).check(matches(isDisplayed()))
+      .check(
+        matches(withText("9"))
+      )
+  }
+
+  fun testTextInputInteractionView_withNoInputText_hasCorrectPendingAnswerType() {
+    val activityScenario = ActivityScenario.launch(InputInteractionViewTestActivity::class.java)
+    activityScenario.onActivity { activity ->
+      val textAnswerRetriever =
+        activity.findViewById(R.id.test_text_input_interaction_view) as TextInputInteractionView
+      assertThat(textAnswerRetriever.getPendingAnswer()).isInstanceOf(InteractionObject::class.java)
+      assertThat(textAnswerRetriever.getPendingAnswer().normalizedString).isEqualTo("")
```

Comment:
You might be able to do assertThat(...).isEmpty() which reads slightly better.

Reply:
resolved